### PR TITLE
fix MyComponents and add Sw360Servlet class

### LINKED ISSUE #56: Fix MyComponentsPortlet
improve performance and show more releases

### PR DESCRIPTION
**Pull request by andi8086**
closes #56

### CODE REVIEW

File: backend/src-common/src/main/java/com/siemens/sw360/projects/Sw360ThriftServlet.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import static org.apache.log4j.Logger.getLogger;
+
+/**
+ * Created by andreas on 27.11.15.
+ */
+public class Sw360ThriftServlet extends TServlet {
```
Comment (maximilianhuber) [inline backend/src-common/src/main/java/com/siemens/sw360/projects/Sw360ThriftServlet.java]: Shouldn't that be `Sw360ThriftServlet`? I thougt one of the goals of this PR was, to replace all `TServlet`'s

**Comment by mcjaeger** [comment]
I am not reviewing this but in the current status, the mycomponents portlet does not seem to run at all: the listing stays always empty. regardless of existing components or creating new ones. see screenshot.
screen shot 2015-12-13 at 21 13 20

**Comment by heydenreich** [comment]
Actually, it works if you start with an empty data base and add components
yourself. But I also have problems when starting with the test database we
use. Therefore I didn't merge so far. (I also don't know the problem either...)



-- 
Dr. Birgit Heydenreich * birgit.heydenreich@tngtech.com * +49-174-3410216
TNG Technology Consulting GmbH, Betastr. 13a, 85774 Unterföhring
Geschäftsführer: Henrik Klagges, Christoph Stock, Dr. Robert Dahlke
Sitz: Unterföhring * Amtsgericht München * HRB 135082

Am 13.12.2015 um 21:14 schrieb Michael:

File: backend/src-common/src/main/java/com/siemens/sw360/projects/Sw360ThriftServlet.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package com.siemens.sw360.projects;
```
Comment (mcjaeger) [inline backend/src-common/src/main/java/com/siemens/sw360/projects/Sw360ThriftServlet.java]: sorry mate, file header missing.

File: frontend/sw360-portlets/src/main/webapp/html/homepage/mycomponents/view.jsp [hunk 3]
```diff
@@ -27,8 +27,6 @@
```
Comment (mcjaeger) [inline frontend/sw360-portlets/src/main/webapp/html/homepage/mycomponents/view.jsp]: file header at top of file please

**Comment by mcjaeger** [comment]
I can confirm that it works

Please correct the two occurrences of missing file headers or file header after the first three lines of XML.

**Comment by heydenreich** [comment]
Does this mean that we should merge?
Did you try with a full db or an empty one?

-- 
Dr. Birgit Heydenreich * birgit.heydenreich@tngtech.com * +49-174-3410216
TNG Technology Consulting GmbH, Betastr. 13a, 85774 Unterföhring
Geschäftsführer: Henrik Klagges, Christoph Stock, Dr. Robert Dahlke
Sitz: Unterföhring * Amtsgericht München * HRB 135082

Am 14.12.2015 um 22:05 schrieb Michael:

**Comment by mcjaeger** [comment]
please correct the header issues before and then merge.

**Comment by mcjaeger** [comment]
OK, saw Andreas e-mail now. Please do not merge for the first, because I am not getting it.

**Pull request merged**